### PR TITLE
Maven Build Support

### PR DESCRIPTION
**Pull request by secondsun**
This PR adds the ability to generate maven artifacts to the core SDK.

# How to Test

from the root of the repository you should run `./gradlew install`.  The build should complete without errors and you should have in your loval maven repository (ex ~/.m2/repository) two new artifacts: "org.aerogear.core" and "org.aerogear.keycloak-module".

This PR includes bits from #4, but once that is merged this can be rebased and merged.

### CODE REVIEW

File: core/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+POM_NAME=AeroGear Android Pipe
```
Comment (wtrocki) [inline core/gradle.properties]: Change name here

**Comment by wtrocki** [comment]
@secondsun Verified changes. It build with success. 
I can reference local maven files in sample android project and it builds just fine.

**Comment by secondsun** [comment]
@wtrocki Thanks, that class was supposed to be in the PoC branch still, good catch.

**Comment by secondsun** [comment]
@wtrocki @wei-lee I've applied your suggested changes.

**Comment by secondsun** [comment]
Landed @https://github.com/aerogear/aerogear-android-sdk/commit/c4f7651d138142607591f54bdd4c51eec28f2305

**Comment by secondsun** [comment]
@wei-lee @wtrocki Merged, but github didn't notice the merge.

**Pull request closed without merging**